### ISSUE TITLE
Image up to date but not really

### ISSUE THREAD

**Issue by ShutdownRepo** [ShutdownRepo/Exegol#97]
Having an issue with `exegol update`, see screenshot below.
![image](https://user-images.githubusercontent.com/40902872/174433558-f78832c6-02eb-47f0-861e-20e6d7f7123d.png)

Below are debug command outputs.

```
$ exegol info -vvv                
[*] Exegol is currently in version v4.0.3b1
[D] Pip installation: Off 🪓                                                                                                                                                                                                                                                              
[D] Git source installation: On ✔                                                                                                                                                                                                                                                         
[D] Host OS: Linux                                                                                                                                                                                                                                                                        
[D] Arch: amd64                                                                                                                                                                                                                                                                           
[D] Docker desktop: Off 🪓                                                                                                                                                                                                                                                                
[D] Shell type: Linux                                                                                                                                                                                                                                                                     
[D] Loading git at /opt/projects/Exegol                                                                                                                                                                                                                                                   
[D] Repo path: /opt/projects/Exegol/.git                                                                                                                                                                                                                                                  
[D] Git repository successfully loaded                                                                                                                                                                                                                                                    
[D] Fetch flags : 4                                                                                                                                                                                                                                                                       
[D] Fetch note :                                                                                                                                                                                                                                                                          
[D] Fetch old commit : None                                                                                                                                                                                                                                                               
[D] Fetch remote path : dev                                                                                                                                                                                                                                                               
[D] HEAD UP TO DATE flag detected                                                                                                                                                                                                                                                         

[V] Listing user configurations                                                                                                                                                                                                                                                           

🧠 User configurations                                     
┌─────────────────────────────────────────────────────────┐
│ Private workspace: /home/cbromberg/.exegol/workspaces   │
│ Exegol resources: /opt/projects/Exegol/exegol-resources │
│ My resources: /home/cbromberg/.exegol/my-resources      │
│ Auto-check updates: On ✔                                │
└─────────────────────────────────────────────────────────┘

[V] Listing git repositories                                                                                                                                                                                                                                                              
[D] Loading git at /opt/projects/Exegol                                                                                                                                                                                                                                                   
[D] Repo path: /opt/projects/Exegol/.git                                                                                                                                                                                                                                                  
[D] Git repository successfully loaded                                                                                                                                                                                                                                                    
[V] Git wrapper init submodules                                                                                                                                                                                                                                                           
[D] Git submodule repository detected                                                                                                                                                                                                                                                     
[D] Loading git at /opt/projects/Exegol/exegol-docker-build                                                                                                                                                                                                                               
[D] Repo path: /opt/projects/Exegol/.git/modules/sources                                                                                                                                                                                                                                  
[D] Git repository successfully loaded                                                                                                                                                                                                                                                    
[D] Git submodule repository detected                                                                                                                                                                                                                                                     
[D] Loading git at /opt/projects/Exegol/exegol-resources                                                                                                                                                                                                                                  
[D] Repo path: /opt/projects/Exegol/.git/modules/exegol-resources                                                                                                                                                                                                                         
[D] Git repository successfully loaded                                                                                                                                                                                                                                                    
[D] Fetch flags : 4                                                                                                                                                                                                                                                                       
[D] Fetch note :                                                                                                                                                                                                                                                                          
[D] Fetch old commit : None                                                                                                                                                                                                                                                               
[D] Fetch remote path : dev                                                                                                                                                                                                                                                               
[D] HEAD UP TO DATE flag detected                                                                                                                                                                                                                                                         
[D] Fetch flags : 4                                                                                                                                                                                                                                                                       
[D] Fetch note :                                                                                                                                                                                                                                                                          
[D] Fetch old commit : None                                                                                                                                                                                                                                                               
[D] Fetch remote path : dev                                                                                                                                                                                                                                                               
[D] HEAD UP TO DATE flag detected                                                                                                                                                                                                                                                         
[D] Fetch flags : 4                                                                                                                                                                                                                                                                       
[D] Fetch note :                                                                                                                                                                                                                                                                          
[D] Fetch old commit : None                                                                                                                                                                                                                                                               
[D] Fetch remote path : main                                                                                                                                                                                                                                                              
[D] HEAD UP TO DATE flag detected                                                                                                                                                                                                                                                         

🐙 Project modules                         
┌───────────┬────────────┬────────────────┐
│ Name      │ Status     │ Current branch │
├───────────┼────────────┼────────────────┤
│ Wrapper   │ Up to date │ dev            │
│ Images    │ Up to date │ dev            │
│ Resources │ Up to date │ main           │
└───────────┴────────────┴────────────────┘

[D] Fetching remote image tags, digests and sizes                                                                                                                                                                                                                                         
[D] Fetching information from: https://hub.docker.com/v2/repositories/nwodtuhs/exegol/tags?page_size=20                                                                                                                                                                                   
[D] └── nightly → (remote) sha256:415150be34aa33bc3967a1f4b93d1173f61c51b98e1880a046b11d4193cb377a                                                                                                                                                                                        
[D] └── full-2.0.2      → (remote) sha256:7d4a12cc5a1e9844f133c8888ed35c3aaceceed1911fb7aace946a8ffe77a232                                                                                                                                                                                
[D] └── full    → (remote) sha256:7d4a12cc5a1e9844f133c8888ed35c3aaceceed1911fb7aace946a8ffe77a232                                                                                                                                                                                        
[D] └── light-2.0.2     → (remote) sha256:630b085d3d8f4e077844ea9b530c4a5856d2db9f7abdcf5ece2ae87fadc80d4d                                                                                                                                                                                
[D] └── light   → (remote) sha256:630b085d3d8f4e077844ea9b530c4a5856d2db9f7abdcf5ece2ae87fadc80d4d                                                                                                                                                                                        
[D] └── ad-2.0.2        → (remote) sha256:a141d1c4e4f50d481a0ab8f2a4d60161c4a099f388485e6f1f8042ed66065611                                                                                                                                                                                
[D] └── ad      → (remote) sha256:a141d1c4e4f50d481a0ab8f2a4d60161c4a099f388485e6f1f8042ed66065611                                                                                                                                                                                        
[D] └── web-2.0.2       → (remote) sha256:f1a210d22838306b14fbd0afcfda51103464239cfb75b752f19bf1b3a8e098be                                                                                                                                                                                
[D] └── web     → (remote) sha256:f1a210d22838306b14fbd0afcfda51103464239cfb75b752f19bf1b3a8e098be                                                                                                                                                                                        
[D] └── osint-2.0.2     → (remote) sha256:d63405881d637711ac2f0a09a321e84ad9caae743cb3b0acc7e9f6ff2558a668                                                                                                                                                                                
[D] └── osint   → (remote) sha256:d63405881d637711ac2f0a09a321e84ad9caae743cb3b0acc7e9f6ff2558a668                                                                                                                                                                                        
[D] └── full-2.0.1      → (remote) sha256:8ff1870bd0dc5c7d79772454b67651d79a23f2b6ad15ab490f00822f6ee324f1                                                                                                                                                                                
[D] └── ad-2.0.1        → (remote) sha256:55e5271e6d8a73d745c3200ce4cb0298b4063e20dfd9f762fc506ca359805d51                                                                                                                                                                                
[D] └── web-2.0.1       → (remote) sha256:ca9c34fb433797ee51199e75b0cebc051fde9b1fe837a03ca5b286478cc05250                                                                                                                                                                                
[D] └── light-2.0.1     → (remote) sha256:9e7f1228f18ec1cc687d2e45fe6ca50013dc30a2f2b0ff4a9cd2bfcbb468dc5d                                                                                                                                                                                
[D] └── osint-2.0.1     → (remote) sha256:7ea562eb869ccc1b729e6468af151b0e4f3d1aeb739baf5d9f20207c5f8b104a                                                                                                                                                                                
[D] └── web-2.0.0       → (remote) sha256:f74fcc92b8e3f31f45e4141fb9eb7f9e85c29e61785eda85b2aa989c56d94571                                                                                                                                                                                
[D] └── light-2.0.0     → (remote) sha256:18fc89446474dfd8ceb2951df41baa571447e4eb6ea7df1b127d8cd6caea222c                                                                                                                                                                                
[D] └── osint-2.0.0     → (remote) sha256:e5e1832f98ce7cc69b9317bab5a9374e2a9d3c0489c36db6572c6ca4387d86b1                                                                                                                                                                                
[D] └── full-2.0.0b5    → (remote) sha256:252afb0667c3d51670d53fdfb53550d71fc7a4d45c4f326b158ff346f0bb0b14                                                                                                                                                                                
[D] Fetching information from: https://hub.docker.com/v2/repositories/nwodtuhs/exegol/tags?page=2&page_size=20                                                                                                                                                                            
[D] └── ad-2.0.0b5      → (remote) sha256:42ac024f403e51dbd5d500afad3750b7666456fa22348740412071ec90481108                                                                                                                                                                                
[D] └── web-2.0.0b5     → (remote) sha256:cc4966d12043bc6a8d98647ab185eef1d157b0bc9103d5ec2cebb3b6e91b98c1                                                                                                                                                                                
[D] └── light-2.0.0b5   → (remote) sha256:b0409163fc61ac9e987843894c9ac53bf117bedce3f36f019115c668fc0c5123                                                                                                                                                                                
[D] └── osint-2.0.0b5   → (remote) sha256:2f693307e10d40785ec0724ccc8b28ef7b7ca4ff6efd21c66da71d0af7d432cd                                                                                                                                                                                
[D] Fetching local image tags, digests (and other attributes)                                                                                                                                                                                                                             
[D] └── nightly → (remote) sha256:f32095a44b8702b9030aba26750e89aace61216cec8662c35f3bc921245ad904                                                                                                                                                                                        
[D] == Loading container : exegol-ec2                                                                                                                                                                                                                                                     
[D] Parsing envs : DISPLAY=:1                                                                                                                                                                                                                                                             
[D] Adding env DISPLAY=:1                                                                                                                                                                                                                                                                 
[D] Parsing envs : _JAVA_AWT_WM_NONREPARENTING=1                                                                                                                                                                                                                                          
[D] Adding env _JAVA_AWT_WM_NONREPARENTING=1                                                                                                                                                                                                                                              
[D] Parsing envs : QT_X11_NO_MITSHM=1                                                                                                                                                                                                                                                     
[D] Adding env QT_X11_NO_MITSHM=1                                                                                                                                                                                                                                                         
[D] Parsing envs : PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                                                                                                                                                                                                      
[D] Adding env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin                                                                                                                                                                                                          
[D] Capabilities : []                                                                                                                                                                                                                                                                     
[D] Load devices : []                                                                                                                                                                                                                                                                     
[D] Parsing mount : {'Type': 'bind', 'Source': '/etc/localtime', 'Destination': '/etc/localtime', 'Mode': '', 'RW': False, 'Propagation': 'rprivate'}                                                                                                                                     
[D] Parsing mount : {'Type': 'bind', 'Source': '/etc/timezone', 'Destination': '/etc/timezone', 'Mode': '', 'RW': False, 'Propagation': 'rprivate'}                                                                                                                                       
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/cbromberg/.exegol/my-resources', 'Destination': '/my-resources', 'Mode': '', 'RW': True, 'Propagation': 'rprivate'}                                                                                                                 
[D] Parsing mount : {'Type': 'bind', 'Source': '/opt/projects/Exegol/exegol-resources', 'Destination': '/opt/resources', 'Mode': '', 'RW': True, 'Propagation': 'rprivate'}                                                                                                               
[D] Parsing mount : {'Type': 'bind', 'Source': '/tmp/.X11-unix', 'Destination': '/tmp/.X11-unix', 'Mode': '', 'RW': True, 'Propagation': 'rprivate'}                                                                                                                                      
[D] Parsing mount : {'Type': 'bind', 'Source': '/home/cbromberg/Downloads', 'Destination': '/workspace', 'Mode': '', 'RW': True, 'Propagation': 'rprivate'}                                                                                                                               
[D] Loading workspace volume source : /home/cbromberg/Downloads                                                                                                                                                                                                                           
[D] Custom workspace detected                                                                                                                                                                                                                                                             
[D] └── nightly → (remote) sha256:f32095a44b8702b9030aba26750e89aace61216cec8662c35f3bc921245ad904                                                                                                                                                                                        
[V] Listing local and remote Exegol images                                                                                                                                                                                                                                                

🛸 Available images                                                                                    
┌───────────────┬───────────┬───────────────┬──────────────┬──────────────────┬───────────────────────┐
│ Id            │ Image tag │ Download size │ Size on disk │ Build date (UTC) │ Status                │
├───────────────┼───────────┼───────────────┼──────────────┼──────────────────┼───────────────────────┤
│ ffb373e4eef2  │ nightly   │ N/A           │ 26.8GB       │ 29/05/2022 10:42 │ Outdated (v.1efeb9f9) │
│ e9ac88beb24a  │ nightly   │ 12.1GB        │ 26.8GB       │ 13/06/2022 09:10 │ Update available      │
│ Not installed │ full      │ 12.6GB        │ N/A          │ N/A              │ Not installed         │
│ Not installed │ light     │ 4.3GB         │ N/A          │ N/A              │ Not installed         │
│ Not installed │ ad        │ 9.3GB         │ N/A          │ N/A              │ Not installed         │
│ Not installed │ web       │ 7.3GB         │ N/A          │ N/A              │ Not installed         │
│ Not installed │ osint     │ 3.1GB         │ N/A          │ N/A              │ Not installed         │
└───────────────┴───────────┴───────────────┴──────────────┴──────────────────┴───────────────────────┘

[V] Listing local Exegol containers                                                                                                                                                                                                                                                       
[*] Number of Exegol containers: 1

👽 Available containers                                                                                                                                                                                                                           
┌────────────┬───────────────┬─────────┬───────────────────────────────┬─────────────────────────┬─────────────────────────────────────────────────────────┬─────────┬───────┬───────────────────────────────────────────────────────────────────┐
│ Id         │ Container tag │ State   │ Image tag                     │ Configurations          │ Mounts                                                  │ Devices │ Ports │ Envs                                                              │
├────────────┼───────────────┼─────────┼───────────────────────────────┼─────────────────────────┼─────────────────────────────────────────────────────────┼─────────┼───────┼───────────────────────────────────────────────────────────────────┤
│ 8fbb243111 │ ec2           │ Stopped │ nightly (outdated v.1efeb9f9) │ Privileged: Off ✔       │ /etc/localtime ➡ /etc/localtime (RO)                    │         │       │ DISPLAY=:1                                                        │
│            │               │         │                               │ GUI: On ✔               │ /etc/timezone ➡ /etc/timezone (RO)                      │         │       │ _JAVA_AWT_WM_NONREPARENTING=1                                     │
│            │               │         │                               │ Network mode: host      │ /home/cbromberg/.exegol/my-resources ➡ /my-resources    │         │       │ QT_X11_NO_MITSHM=1                                                │
│            │               │         │                               │ Share timezone: On ✔    │ /opt/projects/Exegol/exegol-resources ➡ /opt/resources  │         │       │ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin │
│            │               │         │                               │ Exegol resources: On ✔  │ /tmp/.X11-unix ➡ /tmp/.X11-unix                         │         │       │                                                                   │
│            │               │         │                               │ My resources: On ✔      │ /home/cbromberg/Downloads ➡ /workspace                  │         │       │                                                                   │
│            │               │         │                               │                         │                                                         │         │       │                                                                   │
└────────────┴───────────────┴─────────┴───────────────────────────────┴─────────────────────────┴─────────────────────────────────────────────────────────┴─────────┴───────┴───────────────────────────────────────────────────────────────────┘
```
```
$ exegol update nightly --skip-git -vvv
[D] Running update module                                                                                                                                                                                                                                                                 
[*] Exegol is currently in version v4.0.3b1
[D] Pip installation: Off 🪓                                                                                                                                                                                                                                                              
[D] Git source installation: On ✔                                                                                                                                                                                                                                                         
[D] Host OS: Linux                                                                                                                                                                                                                                                                        
[D] Arch: amd64                                                                                                                                                                                                                                                                           
[D] Docker desktop: Off 🪓                                                                                                                                                                                                                                                                
[D] Shell type: Linux                                                                                                                                                                                                                                                                     
[D] Loading git at /opt/projects/Exegol                                                                                                                                                                                                                                                   
[D] Repo path: /opt/projects/Exegol/.git                                                                                                                                                                                                                                                  
[D] Git repository successfully loaded                                                                                                                                                                                                                                                    
[D] Fetch flags : 4                                                                                                                                                                                                                                                                       
[D] Fetch note :                                                                                                                                                                                                                                                                          
[D] Fetch old commit : None                                                                                                                                                                                                                                                               
[D] Fetch remote path : dev                                                                                                                                                                                                                                                               
[D] HEAD UP TO DATE flag detected                                                                                                                                                                                                                                                         

[D] Fetching remote image tags, digests and sizes                                                                                                                                                                                                                                         
[D] Fetching information from: https://hub.docker.com/v2/repositories/nwodtuhs/exegol/tags?page_size=20                                                                                                                                                                                   
[D] └── nightly → (remote) sha256:415150be34aa33bc3967a1f4b93d1173f61c51b98e1880a046b11d4193cb377a                                                                                                                                                                                        
[D] └── full-2.0.2      → (remote) sha256:7d4a12cc5a1e9844f133c8888ed35c3aaceceed1911fb7aace946a8ffe77a232                                                                                                                                                                                
[D] └── full    → (remote) sha256:7d4a12cc5a1e9844f133c8888ed35c3aaceceed1911fb7aace946a8ffe77a232                                                                                                                                                                                        
[D] └── light-2.0.2     → (remote) sha256:630b085d3d8f4e077844ea9b530c4a5856d2db9f7abdcf5ece2ae87fadc80d4d                                                                                                                                                                                
[D] └── light   → (remote) sha256:630b085d3d8f4e077844ea9b530c4a5856d2db9f7abdcf5ece2ae87fadc80d4d                                                                                                                                                                                        
[D] └── ad-2.0.2        → (remote) sha256:a141d1c4e4f50d481a0ab8f2a4d60161c4a099f388485e6f1f8042ed66065611                                                                                                                                                                                
[D] └── ad      → (remote) sha256:a141d1c4e4f50d481a0ab8f2a4d60161c4a099f388485e6f1f8042ed66065611                                                                                                                                                                                        
[D] └── web-2.0.2       → (remote) sha256:f1a210d22838306b14fbd0afcfda51103464239cfb75b752f19bf1b3a8e098be                                                                                                                                                                                
[D] └── web     → (remote) sha256:f1a210d22838306b14fbd0afcfda51103464239cfb75b752f19bf1b3a8e098be                                                                                                                                                                                        
[D] └── osint-2.0.2     → (remote) sha256:d63405881d637711ac2f0a09a321e84ad9caae743cb3b0acc7e9f6ff2558a668                                                                                                                                                                                
[D] └── osint   → (remote) sha256:d63405881d637711ac2f0a09a321e84ad9caae743cb3b0acc7e9f6ff2558a668                                                                                                                                                                                        
[D] └── full-2.0.1      → (remote) sha256:8ff1870bd0dc5c7d79772454b67651d79a23f2b6ad15ab490f00822f6ee324f1                                                                                                                                                                                
[D] └── ad-2.0.1        → (remote) sha256:55e5271e6d8a73d745c3200ce4cb0298b4063e20dfd9f762fc506ca359805d51                                                                                                                                                                                
[D] └── web-2.0.1       → (remote) sha256:ca9c34fb433797ee51199e75b0cebc051fde9b1fe837a03ca5b286478cc05250                                                                                                                                                                                
[D] └── light-2.0.1     → (remote) sha256:9e7f1228f18ec1cc687d2e45fe6ca50013dc30a2f2b0ff4a9cd2bfcbb468dc5d                                                                                                                                                                                
[D] └── osint-2.0.1     → (remote) sha256:7ea562eb869ccc1b729e6468af151b0e4f3d1aeb739baf5d9f20207c5f8b104a                                                                                                                                                                                
[D] └── web-2.0.0       → (remote) sha256:f74fcc92b8e3f31f45e4141fb9eb7f9e85c29e61785eda85b2aa989c56d94571                                                                                                                                                                                
[D] └── light-2.0.0     → (remote) sha256:18fc89446474dfd8ceb2951df41baa571447e4eb6ea7df1b127d8cd6caea222c                                                                                                                                                                                
[D] └── osint-2.0.0     → (remote) sha256:e5e1832f98ce7cc69b9317bab5a9374e2a9d3c0489c36db6572c6ca4387d86b1                                                                                                                                                                                
[D] └── full-2.0.0b5    → (remote) sha256:252afb0667c3d51670d53fdfb53550d71fc7a4d45c4f326b158ff346f0bb0b14                                                                                                                                                                                
[D] Fetching information from: https://hub.docker.com/v2/repositories/nwodtuhs/exegol/tags?page=2&page_size=20                                                                                                                                                                            
[D] └── ad-2.0.0b5      → (remote) sha256:42ac024f403e51dbd5d500afad3750b7666456fa22348740412071ec90481108                                                                                                                                                                                
[D] └── web-2.0.0b5     → (remote) sha256:cc4966d12043bc6a8d98647ab185eef1d157b0bc9103d5ec2cebb3b6e91b98c1                                                                                                                                                                                
[D] └── light-2.0.0b5   → (remote) sha256:b0409163fc61ac9e987843894c9ac53bf117bedce3f36f019115c668fc0c5123                                                                                                                                                                                
[D] └── osint-2.0.0b5   → (remote) sha256:2f693307e10d40785ec0724ccc8b28ef7b7ca4ff6efd21c66da71d0af7d432cd                                                                                                                                                                                
[D] Fetching local image tags, digests (and other attributes)                                                                                                                                                                                                                             
[D] └── nightly → (remote) sha256:f32095a44b8702b9030aba26750e89aace61216cec8662c35f3bc921245ad904                                                                                                                                                                                        
[*] Updating exegol image : nightly                                                                                                                                                                                                                                                       
[!] This image is already up to date. Skipping. 
```

**Comment by ShutdownRepo** [comment]
Similar issue here with a incremental-versioned image (`ad`)

![image](https://user-images.githubusercontent.com/40902872/174433920-5340e123-575d-4109-a63b-2257badb688c.png)

**Comment by ShutdownRepo** [comment]
It seems the status varies on the action (`info`, `start`)
![image](https://user-images.githubusercontent.com/40902872/174444485-60b3c2b2-c833-4481-92c2-f34f76befeeb.png)

**Comment by Dramelac** [comment]
The parsing error in the start process should be fixed in dev.

I need ore information for the image image problem, i cannot reproduce locally.
I've add more debug log, can you retry an `update nightly -vvv` and paste the output ?
Thank you

**Comment by Dramelac** [comment]
Fixed in dev with commit 0a11d6dfdbf96dee5ecd977c028ae35a02dd9f74
Need some testing but i think it will be ok

**Comment by ShutdownRepo** [comment]
Yay :tada: I can confirm both issues are fixed, well done
